scheduler/roles/slurm/tasks/_user.yml

```diff
@@ -0,0 +1,26 @@
+# Copyright 2025 Dell Inc. or its subsidiaries. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+- name: Create slurm group
+  ansible.builtin.group:
+    name: slurm
+    state: present
+
+- name: Add the user 'slurm' with uid 6001 and a primary group of 'slurm'
+  ansible.builtin.user:
+    name: slurm
+    comment: Slurm User Account
+    uid: "{{ slurm_uid }}"
+    group: slurm
+    create_home: false # TODO: if share path is /home this is needed
```

Comment:
for slurm_share, /home will be the  path. will this fail?

Reply:
in our previous code, create_home variable was not there. why this is required now?
![image](https://github.com/user-attachments/assets/373a43ce-6a41-4b83-91ec-804f2e829ed8)